core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NewLineInValueNotice.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * A value in CSV file has a new line or carriage return.
+ *
+ * <p>This error is usually found when the CSV file does not close double quotes properly, so the
+ * next line is considered as a continuation of the previous line.
+ *
+ * <p>Example. The following file was intended to have fields "f11", "f12", "f21", "f22", but it
+ * actually parses as two fields: "f11", "f12\nf21,\"f22\"".
+ *
+ * <pre>
+ *   f11,"f12
+ *   f21,"f22"
+ * </pre>
+ */
+public class NewLineInValueNotice extends ValidationNotice {
+
+  public NewLineInValueNotice(
+      String filename, long csvRowNumber, String fieldName, String fieldValue) {
+    super(
+        ImmutableMap.of(
+            "filename",
+            filename,
+            "csvRowNumber",
+            csvRowNumber,
+            "fieldName",
+            fieldName,
+            "fieldValue",
+            fieldValue),
+        SeverityLevel.ERROR);
```

Comment:
Ugh - so unfortunately the GTFS reference says:

>Field values may not contain tabs, carriage returns or new lines.

[RFC 2119](https://tools.ietf.org/html/rfc2119) defines "MAY" as optional, but does not address "MAY NOT". 

Microsoft apparently treats "MAY NOT" as a prohibited behavior in their [open office spec](https://docs.microsoft.com/en-us/openspecs/office_standards/ms-oe376/cc313153-88b6-4a89-9fdd-8f21e8d1ffb3), and I think most English speakers would agree that this is the intent here in GTFS. So I think it's ok to treat this is as an ERROR and not a WARNING.

@MobilityData/transit-specs?

To align with RFC 2119 the language in the spec should be updated to use `must` instead of `may`.

Reply:
I am open to any point of view.

Sean, if you wish, I can make it WARNING right now and we can change it to ERROR later. I can always patch it to ERROR at Google since we really suffer from such feeds.

Reply:
I agree: `may not` should be treated as `must not`.